components/textdiff/textdiff.less

```diff
@@ -3,4 +3,12 @@
     padding: 10px 0;
     text-align: center;
   }
+
+  td.patched {
```

Comment:
Maybe move this as well to `staging.less` since it is only used there.

Reply:
Sure, no problem (see 34f036a6dca58fc4f546f006a50751c13bad2ccc).